inst/string.m

```diff
@@ -1,4 +1,5 @@
 ## Copyright (C) 2019 Andrew Janke
+## Copyright (C) 2024 Andreas Bertsatos <abertsatos@biol.uoa.gr>
```

Comment:
Want to add yourself to the `CONTRIBUTORS.md` file, too? Feel free to also include a link to your personal website or GitHub profile there if you'd like.

For that matter, maybe these per-file notices should be changed to say "(C) YYYY The Tablicious Developers", the way the Octave source files just say "(C) YYYY The Octave Project Developers". Source code can get moved around over time; who knows whether your particular changes would stay in this file instead of getting migrated or copied somewhere else? i don't have anything that tracks that.

Reply:
"(C) YYYY The Tablicious Developers" is one option, but I think for packages that usually are a one-person or few-persons work, it is more appropriate to keep individual names. At least, this is how I handled it in the statistics, where there have been quite a few people involved in the past. Although most earlier functions have been heavily updated to provide MATLAB compatibility, I retained previous headers and their corresponding dates so that there is some history embedded in the source file itself rather that the .git history log